docs/development.md

```diff
@@ -67,9 +66,13 @@ db information.
 
 ```
 cd gateway-ha/target/
-java -jar gateway-ha-{{VERSION}}-jar-with-dependencies.jar server ../gateway-ha-config.yml
+java --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED -jar gateway-ha-{{VERSION}}-jar-with-dependencies.jar server ../gateway-ha-config.yml
 ```
 
+If you encounter `module java.base does not "opens java.lang" to unnamed module` error, 
+this may be due to newer version of java.
```

Comment:
Newer than 17? I think I will merge #25 first. And we just document what must be done with 17 ...

Reply:
I tried to say that this error is caused by the jdk as it has higher restrictions on internal JDK modules(since jdk 9 i assume), so user has to add additional parameters.
https://stackoverflow.com/questions/42538750/unable-to-export-a-package-from-java-base-module/42541096#42541096